From: Ilias Tsitsimpis <i.tsitsimpis@gmail.com>
Date: Sat, 10 Oct 2015 19:54:16 +0300
Subject: Escape '-' symbol in manpage

Manpage for tabbed was using '-' symbol without escape which will be
considered by groff as hyphen. This patch escapes the '-' in manpages.
This patch also replaces existing example given by upstream to embed an
app into tabbed with a version which uses the secure temp files using
mktemp.

Author: Vasudev Kamath <kamathvasudev@gmail.com>
Forwarded: no
---
 tabbed/tabbed.1 | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/tabbed/tabbed.1 b/tabbed/tabbed.1
index 0ae29ce..9e3b6f8 100644
--- a/tabbed/tabbed.1
+++ b/tabbed/tabbed.1
@@ -22,19 +22,19 @@ tabbed \- generic tabbed interface
 is a simple tabbed container for applications which support XEmbed. Tabbed
 will then run the provided command with the xid of tabbed as appended
 argument. (See EXAMPLES.) The automatic spawning of the command can be
-disabled by providing the -s parameter. If no command is provided
+disabled by providing the \-s parameter. If no command is provided
 tabbed will just print its xid and run no command.
 .SH OPTIONS
 .TP
 .B \-c
-close tabbed when the last tab is closed. Mutually exclusive with -f.
+close tabbed when the last tab is closed. Mutually exclusive with \-f.
 .TP
 .B \-d
 detaches tabbed from the terminal and prints its XID to stdout.
 .TP
 .B \-f
 fill up tabbed again by spawning the provided command, when the last tab is
-closed. Mutually exclusive with -c.
+closed. Mutually exclusive with \-c.
 .TP
 .B \-h
 will print the usage of tabbed.
@@ -54,7 +54,7 @@ will set the WM_CLASS attribute to
 will set the absolute or relative position of where to start a new tab. When
 .I pos
 is is given without 's' in front it is an absolute position. Then negative
-numbers will be the position from the last tab, where -1 is the last tab.
+numbers will be the position from the last tab, where \-1 is the last tab.
 If 's' is given, then
 .I pos
 is a relative position to the current selected tab. If this reaches the limits
@@ -120,15 +120,15 @@ jumps to nth tab
 .B F11
 Toggle fullscreen mode.
 .SH EXAMPLES
-$ tabbed surf -e
+$ tabbed surf \-e
 .TP
-$ tabbed urxvt -embed
+$ tabbed urxvt \-embed
 .TP
-$ tabbed xterm -into
+$ tabbed xterm \-into
 .TP
-$ $(tabbed -d >/tmp/tabbed.xid); urxvt -embed $(</tmp/tabbed.xid);
+$ xid=$(tabbed \-d);urxvt \-embed $xid
 .TP
-$ tabbed -r 2 st -w '' -e tmux
+$ tabbed \-r 2 st \-w '' \-e tmux
 .SH CUSTOMIZATION
 .B tabbed
 can be customized by creating a custom config.h and (re)compiling the source
